tests/components/media_source/test_models.py

```diff
@@ -37,6 +38,7 @@ async def test_browse_media_as_dict():
     assert item["media_content_id"] == f"{const.URI_SCHEME}{const.DOMAIN}/media"
     assert not item["can_play"]
     assert item["can_expand"]
+    assert item["children_media_class"] == MEDIA_CLASS_MUSIC
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)